openedx-scalable-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,678 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "openedxuser",
+      "metadata": {
+        "description": "Administrator username."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Administrator password."
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "appVmCount" : {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "Number of application virtual machines behind the load balancer."
+      }
+    },
+    "appVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D2_v2",
+      "allowedValues": [
```

Comment:
I notice you use standard storage and VM sizes that go with Standard storage. Any particular reason you don't use premium storage? Despite it's name, it's rather cheap compared to the cost of a VM, and it makes storage reliability and performance significantly better.